app/views/items/_item.html.erb

```diff
@@ -1,5 +1,6 @@
 <% @items.each do |item| %>
   <ul>
-    <li><%= item.name %></li>
+    <li><%= link_to "", @item, method: :delete, class: 'gylphicon glyphicon-ok' %>
```

Comment:
The variable I'm using to represent each item in the collection is `@items` which is set to: `@items = @user.items` in the controller. When I used `@items` as the 2nd argument, I get an error message: `undefined method to_model` within the browser.

I just looked at my controller (I may be wrong here), am I missing a defined variable that calls `Item.all`?